gatsby-node.js

```diff
@@ -22,4 +22,17 @@ exports.onCreateWebpackConfig = ({ actions, plugins }) => {
       },
     },
   })
+  // Silence 'conflicting order' warning for CSS modules.
+  // This is only an issue with regular CSS being imported.
+  if (stage === 'build-javascript' || stage === 'develop') {
+    const config = getConfig()
+    // Get the mini-css-extract-plugin
+    const miniCssExtractPlugin = config.plugins.find((plugin) => plugin.constructor.name === 'MiniCssExtractPlugin')
+    // Set the option here to true.
+    if (miniCssExtractPlugin) {
+      miniCssExtractPlugin.options.ignoreOrder = true
+    }
+    // Update the config.
```

Comment:
I don't think we need this gepetty comments here 😄 , maybe just the first two

Reply:
they are Frami comments haha
https://github.com/decentraland/decentraland-gatsby/blob/master/src/plugins/intl/gatsby-node.ts#L96